### PR TITLE
23w44a commands

### PR DESCRIPTION
**Pull request by apple502j**

### CODE REVIEW

File: mappings/net/minecraft/server/command/FunctionCommand.mapping [hunk 1]
```diff
@@ -41,3 +80,8 @@ CLASS net/minecraft/class_3062 net/minecraft/server/command/FunctionCommand
 			ARG 1 context
 		METHOD method_54282 (Lcom/mojang/datafixers/util/Either;)Ljava/util/Collection;
 			ARG 0 functionOrTag
+	CLASS class_8932 ReturnValueCounter
```
Comment (apple502j) [inline mappings/net/minecraft/server/command/FunctionCommand.mapping]:
```suggestion
	CLASS class_8932 ReturnValueAdder
```

File: mappings/net/minecraft/server/command/ExecuteCommand.mapping [hunk 2]
```diff
@@ -246,6 +255,20 @@ CLASS net/minecraft/class_3050 net/minecraft/server/command/ExecuteCommand
...
 		ARG 1 message
 	METHOD method_54264 enqueueExecutions (Lnet/minecraft/class_8839;Ljava/util/List;Ljava/util/function/Function;Ljava/util/function/IntPredicate;Lcom/mojang/brigadier/context/ContextChain;Lnet/minecraft/class_2487;Lnet/minecraft/class_8855;Lnet/minecraft/class_3050$class_8830;Lnet/minecraft/class_8936;)V
+		ARG 0 baseSource
+		ARG 1 sources
+		ARG 2 functionSourceGetter
+		ARG 3 predicate
+		ARG 4 contextChain
+		ARG 5 args
+		ARG 6 control
+		ARG 7 arg4
```
Comment (YanisBft) [inline mappings/net/minecraft/server/command/ExecuteCommand.mapping]: 😠

**Review by YanisBft — COMMENTED** [review]
Seems good, but it would obviously benefit of some documentation at some point

File: mappings/net/minecraft/server/command/AbstractServerCommandSource.mapping [hunk 3]
```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/class_8839 net/minecraft/server/command/AbstractServerCommandSource
-	METHOD method_54307 withResultStorer (Lnet/minecraft/class_8935;)Lnet/minecraft/class_8839;
-		ARG 1 resultStorer
+	METHOD method_54307 withReturnValueConsumers (Lnet/minecraft/class_8935;)Lnet/minecraft/class_8839;
```
Comment (Shnupbups) [inline mappings/net/minecraft/server/command/AbstractServerCommandSource.mapping]: shouldn't be plural i don't think... typo?

File: mappings/net/minecraft/server/command/ExecuteCommand.mapping (hunk 2 above)
Comment (Shnupbups) [inline mappings/net/minecraft/server/command/ExecuteCommand.mapping]: ?? what's this argument?

**Review by Shnupbups — CHANGES_REQUESTED** [review]

**Comment by Shnupbups** [comment]
also not a blocking comment necessarily but what is a Frame and why is its name so generic?

File: mappings/net/minecraft/server/command/ExecuteCommand.mapping (hunk 2 above)
Comment (apple502j) [reply mappings/net/minecraft/server/command/ExecuteCommand.mapping]: I mapped it with Procyon, perhaps that's why it's messed up.

Comment (apple502j) [reply mappings/net/minecraft/server/command/ExecuteCommand.mapping]:
```suggestion
		ARG 7 flags
```

File: mappings/net/minecraft/server/command/AbstractServerCommandSource.mapping (hunk 3 above)
Comment (apple502j) [reply mappings/net/minecraft/server/command/AbstractServerCommandSource.mapping]:
```suggestion
	METHOD method_54307 withReturnValueConsumer (Lnet/minecraft/class_8935;)Lnet/minecraft/class_8839;
```

**Comment by apple502j** [comment]
I believe a frame is an element of a call stack (which can be controlled by, you guessed it, `frameControl` as seen in the record).

File: mappings/net/minecraft/server/command/ExecuteCommand.mapping [hunk 4]
```diff
@@ -246,6 +255,20 @@ CLASS net/minecraft/class_3050 net/minecraft/server/command/ExecuteCommand
...
 	METHOD method_54264 enqueueExecutions (Lnet/minecraft/class_8839;Ljava/util/List;Ljava/util/function/Function;Ljava/util/function/IntPredicate;Lcom/mojang/brigadier/context/ContextChain;Lnet/minecraft/class_2487;Lnet/minecraft/class_8855;Lnet/minecraft/class_3050$class_8830;Lnet/minecraft/class_8936;)V
+		ARG 0 baseSource
+		ARG 1 sources
+		ARG 2 functionSourceGetter
+		ARG 3 predicate
+		ARG 4 contextChain
+		ARG 5 args
+		ARG 6 control
+		ARG 7 flags
+		ARG 8 flags
```
Comment (Shnupbups) [inline mappings/net/minecraft/server/command/ExecuteCommand.mapping]: ??? that can't be right

Comment (apple502j) [reply mappings/net/minecraft/server/command/ExecuteCommand.mapping]: oh no

File: mappings/net/minecraft/server/command/ExecuteCommand.mapping [hunk 5]
```diff
@@ -246,6 +255,20 @@ CLASS net/minecraft/class_3050 net/minecraft/server/command/ExecuteCommand
...
 		ARG 1 message
 	METHOD method_54264 enqueueExecutions (Lnet/minecraft/class_8839;Ljava/util/List;Ljava/util/function/Function;Ljava/util/function/IntPredicate;Lcom/mojang/brigadier/context/ContextChain;Lnet/minecraft/class_2487;Lnet/minecraft/class_8855;Lnet/minecraft/class_3050$class_8830;Lnet/minecraft/class_8936;)V
+		ARG 0 baseSource
+		ARG 1 sources
+		ARG 2 functionSourceGetter
+		ARG 3 predicate
+		ARG 4 contextChain
+		ARG 5 args
+		ARG 6 control
+		ARG 7 flags
```
Comment (apple502j) [inline mappings/net/minecraft/server/command/ExecuteCommand.mapping]:
```suggestion
		ARG 7 functionNamesGetter
```

**Review by Shnupbups — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**